src/couch_replicator_scheduler_job.erl

```diff
@@ -450,11 +450,8 @@ format_status(_Opt, [_PDict, State]) ->
     [{data, [{"State", state_strip_creds(State)}]}].
 
 
-report_job_error(_Rep, _Error) ->
-    % TODO: handle errors back to job scheduler to let it
-    % decide if needed to backoff and retry and what the back-off schedule
-    % should be
-    ok.
+report_job_error(Rep, Error) ->
+    exit({error, Rep, Error}).
```

Comment:
It'll cause the job to terminate in a non-successful way, which is necessary for exponential backoff.